project-koku/koku
# Handle Authentication.create messages from Platform Sources

Addresses #1195 

AWS and Azure store credential information in the Platform Sources authentication store.  The Authentication information can be stored before or after the Cost Management application is attached to the source.  This change handles this event and updates the Source Authentication information regardless of this order.

**Testing**
1. Add Source, endpoint, attach cost management application, authentication, add S3 bucket.  Verify Koku Provider is created.
2. Add Source, endpoint, authentication, attach cost management application, add S3 bucket.  Verify Koku Provider is created.
3.  Add OCP and Azure Sources make sure they are created with helper script.

**Test Results**
[issue_1195_ut.txt](https://github.com/project-koku/koku/files/3679254/issue_1195_ut.txt)
